docs/simulation/calculations.yaml

```diff
@@ -51,7 +51,44 @@ Economy:
       formula: MGQ * GOLD_TO_CASH_RATE
       MGQ: Money good quanity = total production of goods with money = yes
       GOLD_TO_CASH_RATE: defines.country.GOLD_TO_CASH_RATE
+    Overseas maintenance:
+      See: https://github.com/schombert/Project-Alice/blob/main/docs/rules.md#overseas-penalty
   Production:
+    Resource gathering operation:
+      base workforce: defined in common/production_types.txt as workforce
+      rgo size:
+        formula: floor(1.5*ceil(n_workers_in_state / (base workforce * (1 + terrain rgo size modifier)))
+        n_workers_in_state: Total size of POPs in state where POP type is in employees (common/production_types.txt).
+        note: Calculated when starting a new game.
+      Output:
+        formula: base output * rgo size * (1 + overseas penalty) * throughput from workers * (1 + throughput modifier) * (1 + output from workers) * (1 + output modifier)
```

Comment:
So `throughput from workers` is what makes workers create output (with 0 output when there's 0 throughput causing workers), while `output from workers` boosts output further without generally controlling whether it is 0 or not.

Reply:
Correct.